package.json

```diff
@@ -17,10 +17,11 @@
   "contributors": ["Yuri Shkuro <ys@uber.com>", "Won Jun Jang <wjang@uber.com>"],
   "dependencies": {
     "node-int64": "^0.4.0",
-    "thriftrw": "^3.5.0",
-    "xorshift": "^0.2.0",
     "opentracing": "^0.13.0",
-    "uuid": "^3.2.1"
+    "prom-client": "^11.0.0",
```

Comment:
I don't see this dependency in use in the PR, is it required by using a different node version?

Reply:
Awesome catch! npm now saves packages, by default!

>npm install saves any specified packages into dependencies by default.
>– man npm